MAUI/Maps/getting-started.md

```diff
@@ -577,6 +577,4 @@ this.Content = map;
 
 ![.NET MAUI Maps with tile layer](images/getting-started/getting_started_tile_layer.png)
 
-N> 
-* Get the complete getting started sample from [GitHub](https://github.com/SyncfusionExamples/.NET-MAUI-Maps-Getting-Started) link.
-* You can refer to our [.NET MAUI Maps](https://www.syncfusion.com/maui-controls/maui-maps) feature tour page for its groundbreaking feature representations.
\ No newline at end of file
+N> You can refer to our [.NET MAUI Maps](https://www.syncfusion.com/maui-controls/maui-maps) feature tour page for its groundbreaking feature representations. You can also explore our [.NET MAUI Maps Example](https://github.com/syncfusion/maui-demos/tree/master/MAUI/Maps) that shows you how to render the Maps in .NET MAUI.
```

Comment:
why did you remove this?
<img width="729" alt="image" src="https://github.com/syncfusion-content/maui-docs/assets/61148953/73cb4230-64b5-4b3b-a83f-3c0a12a85b48">